### PR TITLE
UI code style cleanup

### PR DESCRIPTION
**Pull request by levitte**
##### Checklist

- [x] CLA is signed

##### Description of change

Mostly condition check changes.

### CODE REVIEW

**Comment by levitte** [comment]
@richsalz, I think you're interested in this.

File: crypto/ui/ui_lib.c [hunk 1]
```diff
@@ -159,8 +159,8 @@ static int general_allocate_boolean(UI *ui,
     } else if (cancel_chars == NULL) {
         UIerr(UI_F_GENERAL_ALLOCATE_BOOLEAN, ERR_R_PASSED_NULL_PARAMETER);
     } else {
-        for (p = ok_chars; *p; p++) {
-            if (strchr(cancel_chars, *p)) {
+        for (p = ok_chars; *p != '\0'; p++) {
+            if (strchr(cancel_chars, *p) != 0) {
```
Comment (richsalz) [inline crypto/ui/ui_lib.c]: NULL not zero.

File: crypto/ui/ui_lib.c [hunk 2]
```diff
@@ -636,52 +640,58 @@ int UI_method_set_prompt_constructor(UI_METHOD *method,
...
         return -1;
 }
 
-int (*UI_method_get_opener(UI_METHOD *method)) (UI *) {
-    if (method)
+int (*UI_method_get_opener(UI_METHOD *method)) (UI *)
+{
+    if (method != NULL)
         return method->ui_open_session;
     else
```
Comment (richsalz) [inline crypto/ui/ui_lib.c]: remove the else.  or consider the ?: operator.

File: crypto/ui/ui_lib.c [hunk 3]
```diff
@@ -636,52 +640,58 @@ int UI_method_set_prompt_constructor(UI_METHOD *method,
...
         return NULL;
 }
 
-int (*UI_method_get_writer(UI_METHOD *method)) (UI *, UI_STRING *) {
-    if (method)
+int (*UI_method_get_writer(UI_METHOD *method)) (UI *, UI_STRING *)
+{
+    if (method != NULL)
         return method->ui_write_string;
     else
```
Comment (richsalz) [inline crypto/ui/ui_lib.c]: remove the else. or consider the ?: operator.

File: crypto/ui/ui_lib.c [hunk 4]
```diff
@@ -636,52 +640,58 @@ int UI_method_set_prompt_constructor(UI_METHOD *method,
...
         return NULL;
 }
 
-int (*UI_method_get_flusher(UI_METHOD *method)) (UI *) {
-    if (method)
+int (*UI_method_get_flusher(UI_METHOD *method)) (UI *)
+{
+    if (method != NULL)
```
Comment (richsalz) [inline crypto/ui/ui_lib.c]: you know what i'm going to say.

**Review by richsalz — CHANGES_REQUESTED** [review]

**Comment by levitte** [comment]
All done

**Review by richsalz — APPROVED** [review]

**Comment by levitte** [comment]
Merged

**Comment by levitte** [comment]
Merged

**Comment by KjellKod** [comment]
@levitte, I am curious. As I am going through the openssl git history for pull request I only see pull requests that are closed. Your comment implies that this was merged but the pull request shows only that it was closed. 

I'm not sure how git is used in the OpenSSL project but it, at least to me, makes it hard for an outsider to see what pull requests are merged vs just closed. Or am I missing something obvious?

**Comment by levitte** [comment]
@KjellKod github isn't really our main repo, even though we work with PRs.  Furthermore, we add a few lines in all the commits to indicate who reviewed, as well as what pull request they came from.  Have a look at 120fb9e43656e1801c75a4fbb7c178ebec9bac18 as an example.  This particular pull ended up as the mentioned commit for the master branch, e01cee6d8fbba09dc811af91a5e24dab95fc66a1 in 1.1.0 and 59ba83c9897b1ed311aa16cb1f575ffae4650068 in 1.0.2.

**Comment by KjellKod** [comment]
Thanks

**Pull request closed without merging**